atlasdb-timelock-server/build.gradle

```diff
@@ -24,6 +24,7 @@ dependencies {
     compile group: 'io.atomix', name: 'atomix'
     compile group: 'io.atomix.catalyst', name: 'catalyst-netty'
     compile group: 'io.dropwizard', name: 'dropwizard-core'
+    compile "com.fasterxml.jackson.datatype:jackson-datatype-jdk8"
```

Comment:
Let's try to be consistent with specifying the sections (`group` and `name`) out fully.

nit: Also could we move this to the top, to maintain alphabetical order?